tox.ini

```diff
@@ -28,6 +29,13 @@ deps =
 commands =
     python -m flake8 semeio tests setup.py
 
+[testenv:pylint]
```

Comment:
This looks good, my only question is, should be merged with `flake8`, and both run together, mainly from a travis resource standpoint?

Reply:
That could make sense, but should we then also merge it with black? When `libres` is pip-installable the cost of running this separately should be very small though... I'm slightly leaning towards keeping it as is, but I'll let you decide ;)